editions/tw5.com/tiddlers/workingwithtw/TiddlyWiki Privacy and Security.tid

```diff
@@ -27,7 +27,7 @@ Users can employ the same tools that they use to keep any document and photograp
 
 There are also open source alternatives such as [[Syncthing|https://syncthing.net/]] that allow users to synchronise files between their devices without using a third party service.
 
-Users may wish to use additional layers of security. TiddlyWiki in the single file configuration itself offers built-in encryption. It uses an industry standard encryption library to offer AES 128 bit encryption in CCM mode. All the data within the file is encrypted, and cannot be accessed without entering the correct password. The password is never stored in the file, and so if it is lost, the data is lost.
+Users may wish to use additional layers of security. TiddlyWiki in the single file configuration itself offers built-in [[encryption|Encryption]]. It uses an industry standard encryption library to offer AES 128 bit encryption in CCM mode. All the data within the file is encrypted, and cannot be accessed without entering the correct password. The password is never stored in the file, and so if it is lost, the data is lost. Instructions can be found [[here|Encryption]]
```

Comment:
IMO using "here" as a link is sub-optimal. I would like to  "Instructions can be found at: [[Encryption]]" -- IMO the first "encryption" link could be removed then.